bundles/org.openhab.persistence.dynamodb/pom.xml

```diff
@@ -15,118 +15,293 @@
+      <version>[1.0, 2.0)</version>
+      <scope>test</scope>
     </dependency>
-    <!-- https://mvnrepository.com/artifact/com.amazonaws/jmespath-java -->
     <dependency>
-      <groupId>com.amazonaws</groupId>
-      <artifactId>jmespath-java</artifactId>
-      <version>1.11.213</version>
-      <exclusions>
-        <exclusion>
-          <groupId>com.fasterxml.jackson.core</groupId>
-          <artifactId>*</artifactId>
-        </exclusion>
-      </exclusions>
+      <groupId>com.almworks.sqlite4java</groupId>
+      <artifactId>sqlite4java-win32-x86</artifactId>
+      <type>dll</type>
+      <version>[1.0, 2.0)</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.almworks.sqlite4java</groupId>
+      <artifactId>sqlite4java-win32-x64</artifactId>
+      <type>dll</type>
+      <version>[1.0, 2.0)</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.almworks.sqlite4java</groupId>
+      <artifactId>libsqlite4java-osx</artifactId>
+      <type>dylib</type>
+      <version>[1.0, 2.0)</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.almworks.sqlite4java</groupId>
+      <artifactId>libsqlite4java-linux-i386</artifactId>
+      <type>so</type>
+      <version>[1.0, 2.0)</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.almworks.sqlite4java</groupId>
+      <artifactId>libsqlite4java-linux-amd64</artifactId>
+      <type>so</type>
+      <version>[1.0, 2.0)</version>
+      <scope>test</scope>
+    </dependency>
+
+    <!-- -->
+    <!-- -->
+    <!-- SDK (runtime) dependencies -->
+    <!-- -->
+    <!-- -->
+    <!-- -->
+    <!-- -->
+    <!-- NOTE: this list is generated automatically using scripts/fetch_sdk.sh to
+      facilitate easier SDK updates. Do not edit the below manually -->
+    <!-- NOTE 2: all transitive dependencies of AWS SDK are included as direct dependencies of this bundle,
+      since we want to embed them to the bundle. The ones specified in dep.noembedded are not embedded though and not even
+      listed here.
+    -->
+    <dependency>
+      <groupId>com.typesafe.netty</groupId>
+      <artifactId>netty-reactive-streams-http</artifactId>
+      <version>2.0.4</version>
     </dependency>
-    <!-- https://mvnrepository.com/artifact/org.apache.httpcomponents/httpclient -->
     <dependency>
-      <groupId>org.apache.httpcomponents</groupId>
-      <artifactId>httpclient</artifactId>
-      <version>4.5.2</version>
+      <groupId>com.typesafe.netty</groupId>
+      <artifactId>netty-reactive-streams</artifactId>
+      <version>2.0.4</version>
     </dependency>
-    <!-- https://mvnrepository.com/artifact/software.amazon.ion/ion-java -->
     <dependency>
-      <groupId>software.amazon.ion</groupId>
-      <artifactId>ion-java</artifactId>
+      <groupId>org.reactivestreams</groupId>
+      <artifactId>reactive-streams</artifactId>
```

Comment:
Needs to be added as exclusion in `bnd.importpackage`?

Reply:
For what it's worth, it looks like a osgi bundle https://github.com/reactive-streams/reactive-streams-jvm/blob/master/api/build.gradle (see the bnd definition)

Reply:
For consistency, I decided to add `org.reactivestreams` as exclusion in bnd.importpackage. 

Tested the packaging with full addons kar, seems to work.